config/routes.rb

```diff
@@ -18,4 +18,5 @@
   end
 
   resources :users, except: :show
+  resources :ingredients, except: :show
```

Comment:
Add ingredients listing somewhere in the navbar

Reply:
Done